jackal_base/scripts/calibrate_compass

```diff
@@ -2,7 +2,8 @@
 
 TEMPDIR=$(mktemp -d --tmpdir calibrate_compass.XXXX)
 BAG_FILE=${TEMPDIR}/imu_record.bag
-CAL_FILE=${TEMPDIR}/imu_compass.yaml
+CAL_FILE=${TEMPDIR}/mag_config.yaml
+CAL_FINAL_PATH=${ROS_ETC_DIR}/jackal_base
```

Comment:
Should the default here be in a directory which needs sudo to write to? I'm fine with that being where the "factory" calibration resides, but when users recalibrate, it would be nice if a) They didn't need sudo to do it and b) They didn't blast away the "factory" calibration